css/store.css

```diff
@@ -0,0 +1,289 @@
+header {
+  height: 100px;
+  display: flex;
+  position: relative;
+}
+header :nth-child(2) {
```

Comment:
"It's a bad habit of mine to use nth-child without fully considering the potential issues, but I'll make sure to avoid it in the future and look for more reliable solutions."